### ISSUE TITLE
0.9.0-rc1 crashes during `Elixir.Credo.Check.Consistency.SpaceInParentheses`

### ISSUE THREAD

**Issue by justinmorrow** [rrrene/credo#494]
This issue does not occur on Credo 0.8.10. Just posting this issue to help with development of the new release :)

### Environment

* Credo version (`mix credo -v`): 0.9.0-rc1
* Erlang/Elixir version (`elixir -v`): 1.5.2
* Operating system: macOS 10.13.2

### What were you trying to do?
`mix credo.gen.config`
`mix credo`

### Expected outcome
Credo to run successfully.

### Actual outcome
```
Checking 88 source files (this might take a while) ...
Error while running Elixir.Credo.Check.Consistency.SpaceInParentheses
** (EXIT from #PID<0.73.0>) an exception was raised:
    ** (ArgumentError) cannot convert the given list to a string.

To be converted to a string, a list must contain only:

  * strings
  * integers representing Unicode codepoints
  * or a list containing one of these three elements

Please check the given list or call inspect/1 to get the list representation, got:

[:System]

        (elixir) lib/list.ex:742: List.to_string/1
        lib/credo/code/token.ex:243: Credo.Code.Token.Elixir1_6_0.to_col_end/3
        lib/credo/code/token.ex:233: Credo.Code.Token.Elixir1_6_0.convert_to_col_end/3
        (elixir) lib/enum.ex:1826: Enum."-reduce/3-lists^foldl/2-0-"/3
        lib/credo/code/token.ex:130: Credo.Code.Token.Elixir1_6_0.position_tuple_for_quoted_string/3
        lib/credo/code/token.ex:85: Credo.Code.Token.Elixir1_6_0.position/1
        lib/credo/code/token.ex:260: Credo.Code.Token.ElixirPre1_6_0.position/2
        lib/credo/code/interpolation_helper.ex:157: Credo.Code.InterpolationHelper.find_interpolations/2


16:47:29.650 [error] Task #PID<0.300.0> started from #PID<0.73.0> terminating
** (ArgumentError) cannot convert the given list to a string.

To be converted to a string, a list must contain only:

  * strings
  * integers representing Unicode codepoints
  * or a list containing one of these three elements

Please check the given list or call inspect/1 to get the list representation, got:

[:System]

    (elixir) lib/list.ex:742: List.to_string/1
    lib/credo/code/token.ex:243: Credo.Code.Token.Elixir1_6_0.to_col_end/3
    lib/credo/code/token.ex:233: Credo.Code.Token.Elixir1_6_0.convert_to_col_end/3
    (elixir) lib/enum.ex:1826: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/credo/code/token.ex:130: Credo.Code.Token.Elixir1_6_0.position_tuple_for_quoted_string/3
    lib/credo/code/token.ex:85: Credo.Code.Token.Elixir1_6_0.position/1
    lib/credo/code/token.ex:260: Credo.Code.Token.ElixirPre1_6_0.position/2
    lib/credo/code/interpolation_helper.ex:157: Credo.Code.InterpolationHelper.find_interpolations/2
Function: #Function<7.45088847/0 in Credo.Check.Runner.run_checks_that_run_on_all/2>
    Args: []

16:47:29.668 [error] GenServer Credo.Supervisor terminating
** (ArgumentError) cannot convert the given list to a string.

To be converted to a string, a list must contain only:

  * strings
  * integers representing Unicode codepoints
  * or a list containing one of these three elements

Please check the given list or call inspect/1 to get the list representation, got:

[:System]

    (elixir) lib/list.ex:742: List.to_string/1
    lib/credo/code/token.ex:243: Credo.Code.Token.Elixir1_6_0.to_col_end/3
    lib/credo/code/token.ex:233: Credo.Code.Token.Elixir1_6_0.convert_to_col_end/3
    (elixir) lib/enum.ex:1826: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/credo/code/token.ex:130: Credo.Code.Token.Elixir1_6_0.position_tuple_for_quoted_string/3
    lib/credo/code/token.ex:85: Credo.Code.Token.Elixir1_6_0.position/1
    lib/credo/code/token.ex:260: Credo.Code.Token.ElixirPre1_6_0.position/2
    lib/credo/code/interpolation_helper.ex:157: Credo.Code.InterpolationHelper.find_interpolations/2
Last message: {:EXIT, #PID<0.73.0>, {%ArgumentError{message: "cannot convert the given list to a string.\n\nTo be converted to a string, a list must contain only:\n\n  * strings\n  * integers representing Unicode codepoints\n  * or a list containing one of these three elements\n\nPlease check the given list or call inspect/1 to get the list representation, got:\n\n[:System]\n"}, [{List, :to_string, 1, [file: 'lib/list.ex', line: 742]}, {Credo.Code.Token.Elixir1_6_0, :to_col_end, 3, [file: 'lib/credo/code/token.ex', line: 243]}, {Credo.Code.Token.Elixir1_6_0, :convert_to_col_end, 3, [file: 'lib/credo/code/token.ex', line: 233]}, {Enum, :"-reduce/3-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 1826]}, {Credo.Code.Token.Elixir1_6_0, :position_tuple_for_quoted_string, 3, [file: 'lib/credo/code/token.ex', line: 130]}, {Credo.Code.Token.Elixir1_6_0, :position, 1, [file: 'lib/credo/code/token.ex', line: 85]}, {Credo.Code.Token.ElixirPre1_6_0, :position, 2, [file: 'lib/credo/code/token.ex', line: 260]}, {Credo.Code.InterpolationHelper, :find_interpolations, 2, [file: 'lib/credo/code/interpolation_helper.ex', line: 157]}]}}
State: {:state, {:local, Credo.Supervisor}, :one_for_one, [{:child, #PID<0.205.0>, Credo.Service.SourceFileSource, {Credo.Service.SourceFileSource, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileSource]}, {:child, #PID<0.204.0>, Credo.Service.SourceFileLines, {Credo.Service.SourceFileLines, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileLines]}, {:child, #PID<0.203.0>, Credo.Service.SourceFileAST, {Credo.Service.SourceFileAST, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileAST]}, {:child, #PID<0.202.0>, Credo.Service.SourceFileScopes, {Credo.Service.SourceFileScopes, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileScopes]}, {:child, #PID<0.201.0>, Credo.Service.Commands, {Credo.Service.Commands, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.Commands]}, {:child, #PID<0.200.0>, Credo.CLI.Output.Shell, {Credo.CLI.Output.Shell, :start_link, []}, :permanent, 5000, :worker, [Credo.CLI.Output.Shell]}], :undefined, 3, 5, [], 0, Supervisor.Default, {[{Credo.CLI.Output.Shell, {Credo.CLI.Output.Shell, :start_link, []}, :permanent, 5000, :worker, [Credo.CLI.Output.Shell]}, {Credo.Service.Commands, {Credo.Service.Commands, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.Commands]}, {Credo.Service.SourceFileScopes, {Credo.Service.SourceFileScopes, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileScopes]}, {Credo.Service.SourceFileAST, {Credo.Service.SourceFileAST, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileAST]}, {Credo.Service.SourceFileLines, {Credo.Service.SourceFileLines, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileLines]}, {Credo.Service.SourceFileSource, {Credo.Service.SourceFileSource, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileSource]}], [strategy: :one_for_one]}}
```

**Comment by rrrene** [comment]
@justinmorrow Thx for providing this feedback!

It seems like Credo is breaking on a line with a call involving the `System` module. 

Could you contribute any more inside into this?

**Comment by cdesch** [comment]
I got the same error using Credo `0.9.0-rc3` with elixir `1.6.1`

```
cjs-MBP15:faker cj$  mix credo --strict 
Checking 50 source files ...
Error while running Elixir.Credo.Check.Consistency.SpaceInParentheses
** (EXIT from #PID<0.73.0>) an exception was raised:
    ** (Protocol.UndefinedError) protocol String.Chars not implemented for {:sigil, {33, 44, nil}, 115, ["  "], [], "("}
        (elixir) lib/string/chars.ex:3: String.Chars.impl_for!/1
        (elixir) lib/string/chars.ex:22: String.Chars.to_string/1
        lib/credo/code/token.ex:243: Credo.Code.Token.Elixir1_6_0.to_col_end/3
        lib/credo/code/token.ex:237: Credo.Code.Token.Elixir1_6_0.convert_to_col_end/3
        (elixir) lib/enum.ex:1899: Enum."-reduce/3-lists^foldl/2-0-"/3
        lib/credo/code/token.ex:130: Credo.Code.Token.Elixir1_6_0.position_tuple_for_quoted_string/3
        lib/credo/code/token.ex:85: Credo.Code.Token.Elixir1_6_0.position/1
        lib/credo/code/interpolation_helper.ex:157: Credo.Code.InterpolationHelper.find_interpolations/2


00:12:16.153 [error] Task #PID<0.192.0> started from #PID<0.73.0> terminating
** (Protocol.UndefinedError) protocol String.Chars not implemented for {:sigil, {33, 44, nil}, 115, ["  "], [], "("}
    (elixir) lib/string/chars.ex:3: String.Chars.impl_for!/1
    (elixir) lib/string/chars.ex:22: String.Chars.to_string/1
    lib/credo/code/token.ex:243: Credo.Code.Token.Elixir1_6_0.to_col_end/3
    lib/credo/code/token.ex:237: Credo.Code.Token.Elixir1_6_0.convert_to_col_end/3
    (elixir) lib/enum.ex:1899: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/credo/code/token.ex:130: Credo.Code.Token.Elixir1_6_0.position_tuple_for_quoted_string/3
    lib/credo/code/token.ex:85: Credo.Code.Token.Elixir1_6_0.position/1
    lib/credo/code/interpolation_helper.ex:157: Credo.Code.InterpolationHelper.find_interpolations/2
Function: #Function<7.109245546/0 in Credo.Check.Runner.run_checks_that_run_on_all/2>
    Args: []

00:12:16.158 [error] GenServer Credo.Supervisor terminating
** (Protocol.UndefinedError) protocol String.Chars not implemented for {:sigil, {33, 44, nil}, 115, ["  "], [], "("}
    (elixir) lib/string/chars.ex:3: String.Chars.impl_for!/1
    (elixir) lib/string/chars.ex:22: String.Chars.to_string/1
    lib/credo/code/token.ex:243: Credo.Code.Token.Elixir1_6_0.to_col_end/3
    lib/credo/code/token.ex:237: Credo.Code.Token.Elixir1_6_0.convert_to_col_end/3
    (elixir) lib/enum.ex:1899: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/credo/code/token.ex:130: Credo.Code.Token.Elixir1_6_0.position_tuple_for_quoted_string/3
    lib/credo/code/token.ex:85: Credo.Code.Token.Elixir1_6_0.position/1
    lib/credo/code/interpolation_helper.ex:157: Credo.Code.InterpolationHelper.find_interpolations/2
Last message: {:EXIT, #PID<0.73.0>, {%Protocol.UndefinedError{description: "", protocol: String.Chars, value: {:sigil, {33, 44, nil}, 115, ["  "], [], "("}}, [{String.Chars, :impl_for!, 1, [file: 'lib/string/chars.ex', line: 3]}, {String.Chars, :to_string, 1, [file: 'lib/string/chars.ex', line: 22]}, {Credo.Code.Token.Elixir1_6_0, :to_col_end, 3, [file: 'lib/credo/code/token.ex', line: 243]}, {Credo.Code.Token.Elixir1_6_0, :convert_to_col_end, 3, [file: 'lib/credo/code/token.ex', line: 237]}, {Enum, :"-reduce/3-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 1899]}, {Credo.Code.Token.Elixir1_6_0, :position_tuple_for_quoted_string, 3, [file: 'lib/credo/code/token.ex', line: 130]}, {Credo.Code.Token.Elixir1_6_0, :position, 1, [file: 'lib/credo/code/token.ex', line: 85]}, {Credo.Code.InterpolationHelper, :find_interpolations, 2, [file: 'lib/credo/code/interpolation_helper.ex', line: 157]}]}}
State: {:state, {:local, Credo.Supervisor}, :one_for_one, [{:child, #PID<0.135.0>, Credo.Service.SourceFileSource, {Credo.Service.SourceFileSource, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileSource]}, {:child, #PID<0.134.0>, Credo.Service.SourceFileLines, {Credo.Service.SourceFileLines, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileLines]}, {:child, #PID<0.133.0>, Credo.Service.SourceFileAST, {Credo.Service.SourceFileAST, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileAST]}, {:child, #PID<0.132.0>, Credo.Service.SourceFileScopes, {Credo.Service.SourceFileScopes, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileScopes]}, {:child, #PID<0.131.0>, Credo.Service.Commands, {Credo.Service.Commands, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.Commands]}, {:child, #PID<0.130.0>, Credo.CLI.Output.Shell, {Credo.CLI.Output.Shell, :start_link, []}, :permanent, 5000, :worker, [Credo.CLI.Output.Shell]}], :undefined, 3, 5, [], 0, Supervisor.Default, {:ok, {%{intensity: 3, period: 5, strategy: :one_for_one}, [{Credo.CLI.Output.Shell, {Credo.CLI.Output.Shell, :start_link, []}, :permanent, 5000, :worker, [Credo.CLI.Output.Shell]}, {Credo.Service.Commands, {Credo.Service.Commands, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.Commands]}, {Credo.Service.SourceFileScopes, {Credo.Service.SourceFileScopes, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileScopes]}, {Credo.Service.SourceFileAST, {Credo.Service.SourceFileAST, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileAST]}, {Credo.Service.SourceFileLines, {Credo.Service.SourceFileLines, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileLines]}, {Credo.Service.SourceFileSource, {Credo.Service.SourceFileSource, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileSource]}]}}}

```

I posted in inssue #469

**Comment by justinmorrow** [comment]
@rrrene This is on my todo list. Expect me to get back to you by the end of this week.

**Comment by asummers** [comment]
Several other checks also throw an error in the same place: `Credo.Check.Readability.TrailingWhiteSpace` and `Credo.Check.Readability.SpaceAfterCommas`.

Temporary workaround:
```
   {Credo.Check.Readability.TrailingWhiteSpace, false},
   {Credo.Check.Consistency.SpaceInParentheses, false},
   {Credo.Check.Readability.SpaceAfterCommas, false}, 
```

**Comment by justinmorrow** [comment]
**TL;DR:** it just looks like the `to_col_end/3` in the `Elixir1_6_0` module is being called even when the Elixir version is 1.5.9, resulting in a crash.

@rrrene Sorry for the slow response. Got a little bogged down with work.

Anyways, looks like we're crashing here in `lib/credo/code/token.ex:243`
```
    def to_col_end(col_start, value, add \\ 0) do
      col_start + String.length(to_string(value)) + add
    end
```

This function is within the module `Elixir1_6_0`. You'll notice I was running Elixir 1.5.2 when I had this issue. I tried running in the same environment just upgrading the version to 1.6.1. Everything seemed to work fine with no error. No other changes were necessary for the code to compile, so I'm fairly confident this is it.

**Comment by rrrene** [comment]
I just released `0.9.0-rc4`. Could you check if the new version addresses the problem?

**Comment by asummers** [comment]
No longer errors on my end.

**Comment by justinmorrow** [comment]
@rrrene 
Looks like the same issue still on both `0.9.0-rc4` and `0.9.0-rc6`:
```
mix credo
Ignoring an undefined check: Credo.Check.Warning.NameRedeclarationByAssignment
Ignoring an undefined check: Credo.Check.Warning.NameRedeclarationByCase
Ignoring an undefined check: Credo.Check.Warning.NameRedeclarationByDef
Ignoring an undefined check: Credo.Check.Warning.NameRedeclarationByFn
Checking 88 source files (this might take a while) ...
Error while running Elixir.Credo.Check.Consistency.SpaceInParentheses
** (EXIT from #PID<0.73.0>) an exception was raised:
    ** (ArgumentError) cannot convert the given list to a string.

To be converted to a string, a list must contain only:

  * strings
  * integers representing Unicode codepoints
  * or a list containing one of these three elements

Please check the given list or call inspect/1 to get the list representation, got:

[:System]

        (elixir) lib/list.ex:742: List.to_string/1
        lib/credo/code/token.ex:217: Credo.Code.Token.PositionHelper.to_col_end/3
        lib/credo/code/token.ex:197: Credo.Code.Token.PositionHelper.convert_to_col_end/3
        (elixir) lib/enum.ex:1829: Enum."-reduce/3-lists^foldl/2-0-"/3
        lib/credo/code/token.ex:128: Credo.Code.Token.PositionHelper.position_tuple_for_quoted_string/3
        lib/credo/code/token.ex:84: Credo.Code.Token.PositionHelper.position/1
        lib/credo/code/token.ex:235: Credo.Code.Token.ElixirPre1_6_0.position/2
        lib/credo/code/interpolation_helper.ex:155: Credo.Code.InterpolationHelper.find_interpolations/2


12:36:03.504 [error] Task #PID<0.304.0> started from #PID<0.73.0> terminating
** (ArgumentError) cannot convert the given list to a string.

To be converted to a string, a list must contain only:

  * strings
  * integers representing Unicode codepoints
  * or a list containing one of these three elements

Please check the given list or call inspect/1 to get the list representation, got:

[:System]

    (elixir) lib/list.ex:742: List.to_string/1
    lib/credo/code/token.ex:217: Credo.Code.Token.PositionHelper.to_col_end/3
    lib/credo/code/token.ex:197: Credo.Code.Token.PositionHelper.convert_to_col_end/3
    (elixir) lib/enum.ex:1829: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/credo/code/token.ex:128: Credo.Code.Token.PositionHelper.position_tuple_for_quoted_string/3
    lib/credo/code/token.ex:84: Credo.Code.Token.PositionHelper.position/1
    lib/credo/code/token.ex:235: Credo.Code.Token.ElixirPre1_6_0.position/2
    lib/credo/code/interpolation_helper.ex:155: Credo.Code.InterpolationHelper.find_interpolations/2
Function: #Function<7.97210212/0 in Credo.Check.Runner.run_checks_that_run_on_all/2>
    Args: []

12:36:03.522 [error] GenServer Credo.Supervisor terminating
** (ArgumentError) cannot convert the given list to a string.

To be converted to a string, a list must contain only:

  * strings
  * integers representing Unicode codepoints
  * or a list containing one of these three elements

Please check the given list or call inspect/1 to get the list representation, got:

[:System]

    (elixir) lib/list.ex:742: List.to_string/1
    lib/credo/code/token.ex:217: Credo.Code.Token.PositionHelper.to_col_end/3
    lib/credo/code/token.ex:197: Credo.Code.Token.PositionHelper.convert_to_col_end/3
    (elixir) lib/enum.ex:1829: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/credo/code/token.ex:128: Credo.Code.Token.PositionHelper.position_tuple_for_quoted_string/3
    lib/credo/code/token.ex:84: Credo.Code.Token.PositionHelper.position/1
    lib/credo/code/token.ex:235: Credo.Code.Token.ElixirPre1_6_0.position/2
    lib/credo/code/interpolation_helper.ex:155: Credo.Code.InterpolationHelper.find_interpolations/2
Last message: {:EXIT, #PID<0.73.0>, {%ArgumentError{message: "cannot convert the given list to a string.\n\nTo be converted to a string, a list must contain only:\n\n  * strings\n  * integers representing Unicode codepoints\n  * or a list containing one of these three elements\n\nPlease check the given list or call inspect/1 to get the list representation, got:\n\n[:System]\n"}, [{List, :to_string, 1, [file: 'lib/list.ex', line: 742]}, {Credo.Code.Token.PositionHelper, :to_col_end, 3, [file: 'lib/credo/code/token.ex', line: 217]}, {Credo.Code.Token.PositionHelper, :convert_to_col_end, 3, [file: 'lib/credo/code/token.ex', line: 197]}, {Enum, :"-reduce/3-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 1829]}, {Credo.Code.Token.PositionHelper, :position_tuple_for_quoted_string, 3, [file: 'lib/credo/code/token.ex', line: 128]}, {Credo.Code.Token.PositionHelper, :position, 1, [file: 'lib/credo/code/token.ex', line: 84]}, {Credo.Code.Token.ElixirPre1_6_0, :position, 2, [file: 'lib/credo/code/token.ex', line: 235]}, {Credo.Code.InterpolationHelper, :find_interpolations, 2, [file: 'lib/credo/code/interpolation_helper.ex', line: 155]}]}}
State: {:state, {:local, Credo.Supervisor}, :one_for_one, [{:child, #PID<0.209.0>, Credo.Service.SourceFileSource, {Credo.Service.SourceFileSource, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileSource]}, {:child, #PID<0.208.0>, Credo.Service.SourceFileLines, {Credo.Service.SourceFileLines, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileLines]}, {:child, #PID<0.207.0>, Credo.Service.SourceFileAST, {Credo.Service.SourceFileAST, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileAST]}, {:child, #PID<0.206.0>, Credo.Service.SourceFileScopes, {Credo.Service.SourceFileScopes, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileScopes]}, {:child, #PID<0.205.0>, Credo.Service.Commands, {Credo.Service.Commands, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.Commands]}, {:child, #PID<0.204.0>, Credo.CLI.Output.Shell, {Credo.CLI.Output.Shell, :start_link, []}, :permanent, 5000, :worker, [Credo.CLI.Output.Shell]}], :undefined, 3, 5, [], 0, Supervisor.Default, {[{Credo.CLI.Output.Shell, {Credo.CLI.Output.Shell, :start_link, []}, :permanent, 5000, :worker, [Credo.CLI.Output.Shell]}, {Credo.Service.Commands, {Credo.Service.Commands, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.Commands]}, {Credo.Service.SourceFileScopes, {Credo.Service.SourceFileScopes, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileScopes]}, {Credo.Service.SourceFileAST, {Credo.Service.SourceFileAST, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileAST]}, {Credo.Service.SourceFileLines, {Credo.Service.SourceFileLines, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileLines]}, {Credo.Service.SourceFileSource, {Credo.Service.SourceFileSource, :start_link, []}, :permanent, 5000, :worker, [Credo.Service.SourceFileSource]}], [strategy: :one_for_one]}}
```

**Comment by rrrene** [comment]
@justinmorrow Any news on this one? Did you get a chance to test with `0.9.0-rc8`?

**Comment by justinmorrow** [comment]
@rrrene Sorry I’ve been traveling for the past week or so. I’ll be able to test this sometime after Tuesday.

**Comment by rrrene** [comment]
Thx for reporting this! :+1: It should be be fixed in the latest version of Credo.

If it is not, please feel free to re-open this issue!
